src/inference/src/dev/core_impl.cpp

```diff
@@ -942,7 +953,11 @@ ov::Any ov::CoreImpl::get_property(const std::string& device_name,
         ov::AnyMap empty_map;
         return coreConfig.get_cache_config_for_device(get_plugin(parsed._deviceName), empty_map)._cacheDir;
     }
-
+    // TODO make more elegant way to clean up internal properties from supported list
```

Comment:
maybe we need to introduce `ov::internal_suppored_properties` ? or even more all internal properties into `ov::internal::<supported_properties | xxx>` or `ov::dev::supported_properties` ?

Reply:
> maybe we need to introduce `ov::internal_suppored_properties` ? or even more all internal properties into `ov::internal::<supported_properties | xxx>` or `ov::dev::supported_properties` ?

It make sense. Thanks.